roles/dns-records/tasks/main.yml

```diff
@@ -50,6 +50,22 @@
   with_items: "{{ groups['infra_hosts'] }}"
   when: hostvars[item]['public_v4'] is defined
 
+- name: "Add public master cluster hostname records to the public A records"
+  set_fact:
+    public_records: "{{ public_records | default([]) + [ { 'type': 'A', 'hostname': hostvars[groups.masters[0]].openshift_master_cluster_public_hostname, 'ip': hostvars[groups.masters[0]].public_v4 } ] }}"
```

Comment:
could it be just the `openshift_master_cluster_public_hostname` inventory var?

Reply:
`openshift_master_cluster_public_hostname` is specified in OSEv3 so I don't think it's exposed in this role directly. But yeah @tzumainn can you test if you could go away with the hostvars business here?

Reply:
Hi! I'm using the value defined in OSEv3.yml as Tomas says, and it looks like just specifying openshift_master_cluster_public_hostname doesn't work in that case.